packages/core/src/common/audioAlert.spec.ts

```diff
@@ -0,0 +1,197 @@
+import { AudioAlertPlayer } from './audioAlertPlayer';
+import { AudioAlert } from './audioAlert';
+import { mostSevere, leastSevere } from './constants';
+
+describe('initializes', () => {
+  const audioAlertPlayer = new AudioAlertPlayer();
+  let audioAlert: AudioAlert;
+
+  it('properly initializes with the given attributes', () => {
+    audioAlert = new AudioAlert({
+      audioAlertPlayer: audioAlertPlayer,
+      isMuted: true,
+      volume: 0.6,
+      severity: 1,
+      audioSrc: 'test string',
+    });
+    expect(audioAlert.audioAlertPlayer).toBe(audioAlertPlayer);
+    expect(audioAlert.isMuted()).toBeTrue();
+    expect(audioAlert.volume).toBe(0.6);
+    expect(audioAlert.severity).toBe(1);
+    expect(audioAlert.audioSrc).toBe('test string');
+  });
+});
+
+describe('play', () => {
+  let audioAlertPlayer: AudioAlertPlayer;
+  let audioAlert: AudioAlert;
+
+  beforeEach(() => {
+    audioAlertPlayer = new AudioAlertPlayer();
+  });
+
+  it('player plays when called on by audio alerts', () => {
+    audioAlert = new AudioAlert({
+      audioAlertPlayer: audioAlertPlayer,
+    });
+    audioAlert.play();
+    expect(audioAlert.audioAlertPlayer.isPlaying()).toBeTrue();
+  });
+
+  it("player doesn't plays when called on by a muted audio alerts", () => {
+    audioAlert = new AudioAlert({
+      audioAlertPlayer: audioAlertPlayer,
+      isMuted: true,
+    });
+    expect(audioAlert.isMuted()).toBeTrue();
+    audioAlert.play();
+    expect(audioAlert.audioAlertPlayer.isPlaying()).toBeFalse();
```

Comment:
Do we need to expose the player implementation for isPlaying() in contrast to how you're hiding it with play()?

Reply:
Nice catch, I forgot to refactor this test when I changed `play()`